contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1170,6 +1165,24 @@ is_tsql_text_ntext_or_image_datatype(Oid oid)
 		(*common_utility_plugin_ptr->is_tsql_image_datatype) (oid);
 }
 
+bool is_tsql_geometry_or_geography_datatype(Oid oid)
+{
+	return (*common_utility_plugin_ptr->is_tsql_geometry_datatype) (oid) ||
+		(*common_utility_plugin_ptr->is_tsql_geography_datatype) (oid);
+}
+
+
+bool is_tsql_rowversion_or_timestamp_datatype(Oid oid)
+{
+	return (*common_utility_plugin_ptr->is_tsql_rowversion_or_timestamp_datatype) (oid);
+}
+
+bool is_tsql_xml_datatype(Oid oid)
+{
+	return (*common_utility_plugin_ptr->is_tsql_xml_datatype) (oid);
+}
```

Comment:
Why do we need yet another wrapper function for this? I would prefer to directly make use of the `common_utility_plugin_ptr`

Reply:
I had created these wrapper function similar to other function to make code cleaner. Removed these in next commit.